api/controllers/builds.go

```diff
@@ -66,6 +66,10 @@ func BuildGet(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 func BuildCreate(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 	build := models.NewBuild(mux.Vars(r)["app"])
 
+	if build.IsLocked() {
```

Comment:
for me, a `lock` would imply actually setting and unsetting a lock. this check seems here to be more around if it is `busy`, `running`, etc. so I think there's a handful of better names than `IsLocked` for this one.  More to consider than a must change.